courses/factories.py

```diff
@@ -72,8 +82,16 @@ class Params:
 class CourseRunEnrollmentFactory(DjangoModelFactory):
     """Factory for CourseRunEnrollment"""
 
+    class Params:
+        has_company_affiliation = Trait(company=SubFactory(CompanyFactory))
+
     user = SubFactory(UserFactory)
     run = SubFactory(CourseRunFactory)
+    company = factory.Maybe(
+        "has_company_affiliation",
```

Comment:
This will always evaluate as `False`, since there's no `has_company_affiliation` attribute on `CourseRunEnrollmentFactory`, you need to use `Faker("pybool")` to randomly pick the yes/no branches.

Reply:
If I switch to `Faker("pybool")`, that will apparently override/ignore `has_company_affiliation` because `test_serialize_program_enrollments[has_company=False]` and `test_serialize_courserun_enrollments[has_company=False]` will start failing about 50% of the time, for example:

>{'company': OrderedDict([('id', 9), ('created_on', '2019-05-17T12:46:56.854422Z'), ('updated_on', '2019-05-17T12:46:56.854464Z'), ('name', 'Leonard PLC')])} != {'company': None}

I think it should be okay for `Company` to be None unless otherwise specified when it's relevant to the test.

Reply:
That sounds like a bug in `factoryboy`, but i'm fine with leaving as-is for now.